src/main/java/by/dragonsurvivalteam/dragonsurvival/common/items/armor/DragonHunterSword.java

```diff
@@ -0,0 +1,25 @@
+package by.dragonsurvivalteam.dragonsurvival.common.items.armor;
+
+import by.dragonsurvivalteam.dragonsurvival.registry.DSEnchantments;
+import by.dragonsurvivalteam.dragonsurvival.registry.DSEquipment;
+import net.minecraft.client.Minecraft;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.world.item.SwordItem;
+import net.minecraft.world.item.enchantment.ItemEnchantments;
+
+public class DragonHunterSword extends SwordItem implements PermanentEnchantmentItem {
+
+    public DragonHunterSword(Properties pProperties) {
+        super(DSEquipment.DRAGON_HUNTER, pProperties);
+    }
+
+    @Override
+    public ItemEnchantments getDefaultEnchantments() {
+        ItemEnchantments.Mutable temp = new ItemEnchantments.Mutable(ItemEnchantments.EMPTY);
+        if (Minecraft.getInstance().level != null && Minecraft.getInstance().level.registryAccess().registry(Registries.ENCHANTMENT).isPresent()) {
+            temp.set(Minecraft.getInstance().level.registryAccess().registry(Registries.ENCHANTMENT).get()
+                    .getHolderOrThrow(DSEnchantments.DRAGONSBANE), 3);
```

Comment:
I was thinking about naming a special sword Dragonsbane, but the enchantment I would just call Bane of Dragons to match up with vanilla names